src/main/java/com/mixpanel/android/mpmetrics/MixpanelActivityLifecycleCallbacks.java

```diff
@@ -39,11 +32,6 @@ public MixpanelActivityLifecycleCallbacks(MixpanelAPI mpInstance, MPConfig confi
 
     @Override
     public void onActivityStarted(Activity activity) {
```

Comment:
can we delete this since it's empty?

Reply:
no, it implements the interface `ActivityLifecycleCallbacks` and it is mandatory to implement all the methods in it.